xml/MAIN.SLEDS.xml

```diff
@@ -121,7 +121,7 @@
   <xi:include href="art_installation-sleds.xml"/>
   <xi:include href="art_modules.xml"/>
   <xi:include os="sles" href="art_virtualization-best-practices.xml"/>
-  <xi:include os="sles" href="art_jeos.xml"/>
+  <xi:include os="sles" href="art_minvm.xml"/>
```

Comment:
Due to the above reasons, you should use `article_minimal_vm.xml`:

```suggestion
  <xi:include os="sles" href="article_minimal_vm.xml"/>
```

Reply:
I don't mind the xml file names as much -- unlike IDs, customers never see them. (Though making sure they're consistent might avoid future confusion of your co-workers.)